data-protocols/ids/ids-api-transfer/src/test/java/org/eclipse/dataspaceconnector/ids/api/transfer/ArtifactRequestControllerTest.java

```diff
@@ -0,0 +1,159 @@
+/*
+ *  Copyright (c) 2021 Microsoft Corporation
```

Comment:
or to Catena-X Consortium ?
```suggestion
 *  Copyright (c) 2021 Copyright Holder (Catena-X Consortium)
```

Reply:
I changed it in both test classes